BridgeCareApp/AspWebApi/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Reflection;
```

Comment:
This can probably be reduced to [assembly: System.Reflection.AssemblyVersion("0.0.*")]

Reply:
Used this